gmso/formats/lammpsdata.py

```diff
@@ -199,3 +209,228 @@ def write_lammpsdata(topology, filename, atom_style='full'):
         # TODO: Write out bonds
         # TODO: Write out angles
         # TODO: Write out dihedrals
+
+
+def read_lammpsdata(filename, atom_style='full', unit_style='real', potential='lj'):
+    """
```

Comment:
For numpydoc style, the single line should start at the end of the triple quotes. 
```suggestion
    """Read in a lammps data file as a GMSO topology
```